test/config-next/orphan-finder.json

```diff
@@ -9,10 +9,10 @@
   },
 
   "saService": {
-    "serverAddresses": ["boulder:9095"],
+    "serverAddresses": ["sa.boulder:9095"],
     "serverIssuerPath": "test/grpc-creds/minica.pem",
-    "clientCertificatePath": "test/grpc-creds/boulder-client/cert.pem",
-    "clientKeyPath": "test/grpc-creds/boulder-client/key.pem",
+    "clientCertificatePath": "test/grpc-creds/sa.boulder/cert.pem",
```

Comment:
Should the `clientCertificatePath` and `clientKeyPath` here be an orphan-finder cert/key instead of the SA cert/key?